metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -122,100 +122,103 @@ endif::[]
-<<configuring-howto-metricbeat>>.
+<<configuring-howto-{beatname_lc}>>.
 
-[[metricbeat-configuration]]
-=== Step 2: Configure Metricbeat
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
 
 include::../../libbeat/docs/shared-configuring.asciidoc[]
 
-Metricbeat uses <<metricbeat-modules,modules>> to collect metrics. You configure
-each module individually. The following example shows the default configuration
-in the `metricbeat.yml` file. The system status module is enabled by default to
-collect metrics about your server, such as CPU usage, memory usage, network IO
-metrics, and process statistics:
-
-[source, shell]
--------------------------------------
-metricbeat.modules:
-- module: system
-  metricsets:
-    - cpu
-    - filesystem
-    - memory
-    - network
-    - process
-  enabled: true
-  period: 10s
-  processes: ['.*']
-  cpu_ticks: false
--------------------------------------
-
-The following example shows how to configure two modules: the system module
-and the Apache HTTPD module:
-
-[source, shell]
--------------------------------------
-metricbeat.modules:
-- module: system
-  metricsets:
-    - cpu
-    - filesystem
-    - memory
-    - network
-    - process
-  enabled: true
-  period: 10s
-  processes: ['.*']
-  cpu_ticks: false
-- module: apache
-  metricsets: ["status"]
-  enabled: true
-  period: 1s
-  hosts: ["http://127.0.0.1"]
--------------------------------------
-
-To configure Metricbeat:
-
-. Define the Metricbeat modules that you want to enable. For each module, specify
-the metricsets that you want to collect. See <<configuring-howto-metricbeat>> for
-more details about configuring modules.
+When you configure {beatname_uc}, you need to specify which
+<<{beatname_lc}-modules,modules>> to run. {beatname_uc} uses modules to collect
+metrics. Each module defines the basic logic for collecting data from a specific
+service, such as Redis or MySQL. A module consists of metricsets that fetch and
+structure the data. Read <<how-{beatname_lc}-works>> to learn more.
+
+To configure {beatname_uc}:
+
+. Enable the modules that you want to run. If you accept the default
+configuration without enabling additional modules, {beatname_uc} collects system
+metrics only.
++
+You can either enable the default module configurations defined in the
+`modules.d` directory (recommended), or add the module configs to the
++{beatname_lc}.yml+ file. The `modules.d` directory contains default
+configurations for all available {beatname_uc} modules. 
++
+The following examples enable the `apache` and `mysql` configs in the
+`modules.d` directory :
++
+*deb and rpm:*
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc} modules enable apache mysql
+----
++
+*mac:*
++
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} modules enable apache mysql
+----
++
+*docker*:
++
+["source","sh",subs="attributes"]
+----
+docker run docker.elastic.co/beats/{beatname_lc}:{version} modules enable apache mysql
```

Comment:
I think this won't work, as docker doesn't store the container status between runs

Reply:
Probably it's tricky, they would need to mount a `modules.d` folder or `metricbeat.yml` from the host,

Reply:
@exekias Yah, that makes sense. I was trying to add platform-specific commands for all the platforms we currently show in the getting started docs, but I didn't think it through very carefully. Since Docker is a special beast, sounds like I just need to point to: 

https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-docker.html

Reply:
@exekias One question, tho...should we mention in the command reference that the modules command won't work with Docker images?

Reply:
I would skip the docker command and only link to the docs as the command alone spins up the container but will not provide the user any useful output.

Reply:
I'm ok if we skip docker here

Reply:
@ruflin I pushed another commit that removes the run example (and points to the full topic about running on docker) because it sounds like it's not useful to run that command. If it looks good, feel free to squash and commit.